Box.V2/Managers/BoxSearchManager.cs

```diff
@@ -63,6 +63,7 @@ public BoxSearchManager(IBoxConfig config, IBoxService service, IBoxConverter co
                                                              
         {
 
+            keyword.ThrowIfNullOrWhiteSpace("keyword");
```

Comment:
I think it should be safe to remove = null, since no one should be using it w/o keyword.

My another question, why this param named keyword instead query to align with our API doc?